config/environments/development.rb

```diff
@@ -163,7 +159,7 @@
   # Second, if MarkUs is configured with REPOSITORY_EXTERNAL_SUBMITS_ONLY
   # set to 'true', you can configure as to where MarkUs should write the
   # Subversion authz file.
-  REPOSITORY_PERMISSION_FILE = REPOSITORY_STORAGE + '/conf'
+  REPOSITORY_PERMISSION_FILE = REPOSITORY_STORAGE + "/conf"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.